share/picongpu/tests/Synchrotron_plugin/lib/validate.py

```diff
@@ -113,17 +133,18 @@ def main(dataPath):
     delta = delta[mask]
     analytical_integrated = np.array(analytical_integrated)[mask]
 
-    x, y = subtract_functions(delta, analytical_integrated, b, a)
-    poorness = np.sum(y) / len(y)  # average error
+    x, y = relative_error_array(delta, analytical_integrated, b, a)
+    poorness = np.sum(y) / len(y)  # average relative error
     poornessBound = 0.1  # 10% error. We want poorness to be less than 10%
     print(f"Poorness: {poorness}")
 
     retValue = int(0) if poorness <= poornessBound else int(1)
+    print(f"Test {'passed' if retValue == 0 else 'failed'}")
 
     sys.exit(retValue)
 
 
 if __name__ == "__main__":
     if len(sys.argv[1:]) != 1:
-        raise SystemExit(f"Usage: {sys.argv[0]} <path_to_simulation_data>")
+        raise ValueError(f"Usage: {sys.argv[0]} <path_to_simulation_data>")
```

Comment:
Just a question: why this change?

Reply:
I guess that was in response to [this comment](https://github.com/ComputationalRadiationPhysics/picongpu/pull/4926#discussion_r1629120358) applying to a different exception in the same script. The change you are referring to will most likely not make any difference at all and one could argue that not doing it might convey the intent a tiny bit better. So, weakly agreeing with you but probably doesn't matter.